Makefile

```diff
@@ -29,7 +29,7 @@ destroy: down ## Bring down the Kafka2Hbase Docker container and services then d
 
 .PHONY: integration
 integration: up ## Run the integration tests in a Docker container
-	docker-compose run --rm integration-test ./gradlew integration
+	docker-compose run --rm integration-test ./gradlew --rerun-tasks integration
```

Comment:
presumably this isn't needed if the `upToDateWhen` thing is used below?

Reply:
Done